src/it/test_coverage_minima/validate.groovy

```diff
@@ -0,0 +1,38 @@
+def checkModule(logText, module, coverageLog) {
```

Comment:
Awesome tests! Thank you.

Reply:
you added the tests, i just duplicated :)